api/types/app.go

```diff
@@ -239,6 +241,10 @@ func (a *AppV3) GetAWSAccountID() string {
 	return a.Metadata.Labels[constants.AWSAccountIDLabel]
 }
 
+func (a *AppV3) GetTeleportVersion() string {
+	return a.Version
+}
```

Comment:
Hmm, I think `a.Version` represents the _resource_ version, not Teleport version. There is also `GetVersion()` method that returns the same.

Applications themselves don't have information about Teleport version. AppServer resources (which represent an application proxied by a particular Teleport agent) do.

So I don't think this method is neede? tctl apps ls already display AppServers IIRC.

Reply:
Yeah this wasnt needed, just a holdover from when I was trying some stuff, I'm using appserver anyway, removed.